test/integration/test-amp-inabox.js

```diff
@@ -131,8 +131,8 @@ describe('inabox', function() {
         <script src="/examples/inabox-tag-integration.js"></script>
         `,
     }, env => {
-      it('should layout amp-img, amp-pixel, ' +
-          'amp-analytics', () => {
+      // TODO(zombifier, #21545): fix this flaky test
+      it.skip('should layout amp-img, amp-pixel, amp-analytics', () => {
```

Comment:
It's the test at line 164 (under 'AMPHTML ads rendered on non-AMP page BTF' that's skipping for Safari) of this file that is flaking, not this one.

Apologies for the confusion; they're all named the same even across multiple files, and only their parents are different.

Reply:
Oh man, I'm acting like a drunk build cop...

Reply:
OK now?

Reply:
It is now, thanks. Maybe I should rename them later on :laughing: